test/spanner.ts

```diff
@@ -512,6 +513,109 @@ describe('Spanner with mock server', () => {
     });
   });
 
+  describe('session-not-found', () => {
+    it('should retry "Session not found" errors on Database.run()', done => {
+      const db = newTestDatabase({
+        min: 0,
+      });
+      spannerMock.setExecutionTime(
+        spannerMock.executeStreamingSql,
+        SimulatedExecutionTime.ofError({
+          code: status.NOT_FOUND,
+          message: 'Session not found',
+        } as MockError)
+      );
+      db.run(selectSql, (err, rows) => {
+        if (err) {
+          assert.fail(err);
+          done();
+          return;
+        }
+        assert.strictEqual(rows!.length, 3);
+        db.getSessions((err, results) => {
+          if (err) {
+            assert.fail(err);
+            done();
+            return;
+          }
+          // The server should have exactly 2 sessions. The first one that was
+          // created was removed from the session pool because of the simulated
```

Comment:
Change to `The first session was removed ...`?

Reply:
Done